src/checks.rs

```diff
@@ -2432,6 +2445,17 @@ impl CheckKind {
                 DebuggerUsingType::Import(name) => format!("Import for `{name}` found"),
             },
             // flake8-tidy-imports
+            CheckKind::BannedApi {
+                name,
+                message,
+                attribute_access,
+            } => {
+                if *attribute_access {
+                    format!("`{name}` is banned: {message} (this may be a false positive)")
```

Comment:
I think claiming that something is the case when it is not is much more confusing than having a disclaimer that the check result may be wrong. Whether or not false-positive cases are rare depends entirely on the code base. The check is not authoritative, it's just an educated guess ... I think the wording of the message should reflect that.

Reply:
I hear you but I just don't think this is a tenable position. There are so many ways to avoid static detection in Python. By this logic, we should be adding a disclaimer like this to almost every check.

Reply:
Fair enough, I removed it :)